### ISSUE TITLE
Correción Desarrollo de la contribución

### ISSUE THREAD

**Issue by estebanolroberto** [estebanolroberto/TFM_SERGIO-ROBERTO-UNIR#54]
Problema: Detalles técnicos sin criterios claros [Roberto]

- [x] Se describen transformaciones como limpieza de texto, tokenización, lematización, pero no se justifica por qué se eligieron determinadas técnicas (por ejemplo, lematización vs stemming).
- [x] Justificar decisiones con base en el tipo de datos analizados (informalidad en Twitter, estructura en noticias, etc.)

Problema: Falta de validación de la calidad de los datos preparados [Roberto]

- [x] No se reporta si hubo verificación de equilibrio de clases, outliers, o problemas de overfitting anticipado por desbalance.
- [x] Añadir métricas básicas sobre la distribución final de clases o emociones tras la limpieza.

Problema: Modelado sin trazabilidad ni explicación de diseño [Sergio]

- [x] Se menciona uso de XGBoost, Random Forest, LSTM y Prophet sin explicar por qué estos modelos fueron elegidos ni cuál es su papel exacto.
- [x] No queda claro si se busca predicción de rango de precios, clasificación de volatilidad, detección de picos, etc.
- [x] Definir claramente el objetivo del modelado (p.ej., clasificación binaria de volatilidad alta/baja).
- [ ] Incluir comparativa tabulada de métricas (accuracy, F1, ROC-AUC) entre modelos. [Comentario Roberto: Creo que esto ya lo tenemos puesto ]

**Comment by estebanolroberto** [comment]
--- Análisis de la colección: memecoins_reddit_sentiment_bert_11_06_25 ---

Descripción estadística de la columna 'score':
count    9971.000000
mean        0.355155
std         0.427710
min         0.000000
25%         0.000000
50%         0.000000
75%         0.750000
max         1.000000
Name: score, dtype: float64

Distribución porcentual de clases:
sentiment_class
neutral     54.4
positive    45.6
Name: proportion, dtype: float64

Conteo absoluto de clases:
sentiment_class
neutral     5445
positive    4565
Name: count, dtype: int64

Número de posibles outliers en 'score' según IQR: 0


--- Análisis de la colección: memecoins_reddit_sentiment_gpt_11_06_25 ---

Descripción estadística de la columna 'score':
count    10010.000000
mean         0.501119
std          0.231755
min          0.000000
25%          0.300000
50%          0.500000
75%          0.750000
max          1.000000
Name: score, dtype: float64

Distribución porcentual de clases:
sentiment_class
positive    99.16
neutral      0.84
Name: proportion, dtype: float64

Conteo absoluto de clases:
sentiment_class
positive    9926
neutral       84
Name: count, dtype: int64

Número de posibles outliers en 'score' según IQR: 0


--- Análisis de la colección: memecoins_reddit_sentiment_vader_11_06_25 ---

Descripción estadística de la columna 'score':
count    10010.000000
mean         0.052591
std          0.456020
min         -0.999600
25%         -0.258775
50%          0.000000
75%          0.381800
max          0.999500
Name: score, dtype: float64

Distribución porcentual de clases:
sentiment_class
positive    35.53
neutral     34.05
negative    30.42
Name: proportion, dtype: float64

Conteo absoluto de clases:
sentiment_class
positive    3557
neutral     3408
negative    3045
Name: count, dtype: int64

Número de posibles outliers en 'score' según IQR: 0

**Comment by sergioddlp** [comment]
esto: 

Incluir comparativa tabulada de métricas (accuracy, F1, ROC-AUC) entre modelos. [Comentario Roberto: Creo que esto ya lo tenemos puesto ]

en principio esta, no?

@estebanolroberto

**Comment by estebanolroberto** [comment]
@sergioddlp Tenemos ya, MSE,RMSE, MAE y R2 -> Tabla 15
